elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -281,6 +281,11 @@ public FilterExpression visit(ComparisonNode node, Class entityType) {
 
             //Coerce arguments to their correct types
             List<Object> values = arguments.stream()
+                    .map(argument ->
```

Comment:
This doesn't look correct.  Take a look at the `equalityExpression` function below.  It needs the asterisks to determine the correct operator.  By stripping them here, when `equalityExpression` is called, it will always map to equals.